docs/rules/index.md

```diff
@@ -5,42 +5,45 @@ layout: doc
 <!-- Note: No pull requests accepted for this file. See README.md in the root directory for details. -->
```

Comment:
Yes, css changes should stay in this PR and a new PR should be opened in main ESLint repository for md changes. Thanks!